fbchat/client.py

```diff
@@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
     def _console(self, msg):
         if self.debug: print(msg)
 
-    def _setttstamp(self):
```

Comment:
Because it's still a function. It is “clean programming”. 

Also, have you seen the Javascript version this is based on?






From: Sid Kapur
Sent: ‎Wednesday‎, ‎6‎ ‎April‎ ‎2016 ‎22‎:‎41
To: carpedm20/fbchat
Cc: Pidgey





In fbchat/client.py:
> @@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
>      def _console(self, msg):
>          if self.debug: print(msg)
>  
> -    def _setttstamp(self):


Why is this a good reason?

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub